.travis.yml

```diff
@@ -1,8 +1,34 @@
+conditions: v1
 language: python
-sudo: required
-dist: xenial
-python: "3.7"
-cache: pip
+python:
+  - "3.6"
 before_install:
 - pip install -r requirements.txt
 script: pre-commit run --all-files
+
+jobs:
+  include:
+  - python: 3.7
+    dist: xenial
+    sudo: true
+
+  - stage: &stage_deploy_name upload new version of python package to PYPI (only for tagged commits)
+    name: Deploy
+    if: tag IS present
+    python: 3.6
+    install: skip
+    script: skip
+    deploy:
+      provider: pypi
+      on:
+        all_branches: true
+        tags: true
+      user: serhii73
```

Comment:
I'd probably have a separate "bot" account with low privileges for the deployment purposes. Also, PyPI will have scoped access tokens quite soon which is very promising.

Reply:
(It's available for local testing btw -> https://github.com/pypa/warehouse/pull/6084)

Reply:
Do you mean [it](https://twitter.com/webKnjaZ/status/1154453110409928704)?

Reply:
Yes, I mean *this*.